frontend-react/src/components/Markdown/MarkdownDirectory.ts

```diff
@@ -0,0 +1,22 @@
+/* Used to instantiate a set of static pages, like BuiltForYouIndex
```

Comment:
I'm on a mission to refactor _out_ the utils files. They tend to become a junk drawer for relatively similar functions a lot of times. That said, I do think it should be either wrapped into a component file or put elsewhere. I think this could easily be merged into the `StaticPageFromDirectories` code since these are the directories it uses to generate the content. Makes sense to me as a module.